pinot-core/src/main/java/org/apache/pinot/core/segment/index/SegmentMetadataImpl.java

```diff
@@ -43,19 +43,19 @@
 import org.apache.commons.configuration.ConfigurationException;
 import org.apache.commons.configuration.PropertiesConfiguration;
 import org.apache.commons.lang.StringEscapeUtils;
-import org.apache.pinot.spi.data.MetricFieldSpec;
-import org.apache.pinot.spi.data.Schema;
 import org.apache.pinot.common.metadata.segment.RealtimeSegmentZKMetadata;
 import org.apache.pinot.common.segment.SegmentMetadata;
 import org.apache.pinot.common.segment.StarTreeMetadata;
-import org.apache.pinot.spi.utils.JsonUtils;
-import org.apache.pinot.spi.utils.TimeUtils;
 import org.apache.pinot.core.indexsegment.generator.SegmentVersion;
 import org.apache.pinot.core.segment.creator.impl.V1Constants;
 import org.apache.pinot.core.segment.creator.impl.V1Constants.MetadataKeys;
 import org.apache.pinot.core.segment.store.SegmentDirectoryPaths;
 import org.apache.pinot.core.startree.v2.StarTreeV2Constants;
 import org.apache.pinot.core.startree.v2.StarTreeV2Metadata;
+import org.apache.pinot.spi.data.MetricFieldSpec;
```

Comment:
no other changes here? why touch this file?

Reply:
The file is auto formatted - had the wrong import order (after spi refactor)

Reply:
you can undo your checkout (or, checkout from master and re-populate it)

Reply:
Since the import order should be fixed and has been fixed, I would like to keep the change as is. Will avoid such changes in the future.